helmfile/charts/harbor/harbor-backup/scripts/harbor-backup.sh

```diff
@@ -42,7 +44,7 @@ s3_upload() {
 
     PATH_TO_BACKUP=s3://${BUCKET_NAME}"/backups/"$(date +%s).sql.gz
```

Comment:
Nit: fix if you want. Same for gcs further down.
```suggestion
    PATH_TO_BACKUP=s3://${BUCKET_NAME}"/backups/"$(date +%s).tgz
```